WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -607,13 +637,20 @@ class PostListViewModel @Inject constructor(
         when (instanceTag) {
             CONFIRM_DELETE_POST_DIALOG_TAG -> localPostIdForTrashDialog = null
             CONFIRM_PUBLISH_POST_DIALOG_TAG -> localPostIdForPublishDialog = null
+            CONFIRM_ON_CONFLICT_LOAD_REMOTE_POST_DIALOG_TAG -> localPostIdForConflictResolutionDialog?.let {
+                localPostIdForConflictResolutionDialog = null
+                loadPostFromLocal(it)
+            }
             else -> throw IllegalArgumentException("Dialog's negative button click is not handled: $instanceTag")
         }
     }
 
     fun onDismissByOutsideTouchForBasicDialog(instanceTag: String) {
-        // Cancel and outside touch dismiss works the same way
-        onNegativeClickedForBasicDialog(instanceTag)
+        // Cancel and outside touch dismiss works the same way for all, except for conflict resolution dialog,
+        // for which tapping outside and actively tapping the "edit local" have different meanings
+        if (instanceTag != CONFIRM_ON_CONFLICT_LOAD_REMOTE_POST_DIALOG_TAG) {
```

Comment:
I guess this is mostly a design question, but would we prefer to make this a non-cancelable dialog instead? I feel like this is a good opportunity to make sure this conflict is resolved by the user. cc @megsfulton

Reply:
My thoughts are that when put under this situation, the user may also prefer to double check on the web before making a decision? Especially given we won't be (for now) allowing them to preview what's on one version and the other. So, forcing them to not have a way to escape would then let them stuck there until they can check on the web. Wdyt?